index.html

```diff
@@ -74,7 +74,7 @@
                     <div class="col-md-8 col-md-offset-2">
                         <img src="planktos-logo.png" alt="" style="width: 20%">
                         <h1 class="brand-heading">Planktos</h1>
-                        <p class="intro-text">Serving websites over bittorrent</p>
+                        <p class="intro-text">Enabling websites to serve their static content over BitTorrent</p>
```

Comment:
We will need to brainstorm this... I'm thinking something along the lines of "Converting Websites into Torrents"